cmake/SetupCompilerOptions.cmake

```diff
@@ -229,8 +229,18 @@ if( BLT_CXX_STD STREQUAL c++98 )
     set(CMAKE_CXX_STANDARD 98)
 elseif( BLT_CXX_STD STREQUAL c++11 )
     set(CMAKE_CXX_STANDARD 11)
+    if (COMPILER_FAMILY_IS_XL)
```

Comment:
I'm all for suggestions on a more universal way to handle this. I believe that we would run into this problem with Intel also on older CMake versions.